tapis-project/tapis-ui
# task/TUI-311 - rework of wizard form

## Overview:

Rework of Wizard wrapper

## Related Github Issues:

- [TUI-311](https://github.com/tapis-project/tapis-ui/issues/311)

## Summary of Changes:

- Wizard wrapper is now also a Formik Wrapper
- Submit button replaced with Skip To End button, moved into Formik DOM
- JobJSON step is now JobSubmission step
- Move generateDefaultValues to generateJobDefaults
- Job steps are now objects, used to generate wizard steps

## Testing Steps:

1. Test bsf v1 and NoExecSystem apps
2. Try changing apps
3. Try changing values in apps
4. Test both continue and skip buttons
5. Check job JSON summaries to ensure changes propagate correctly

## UI Photos:

![image](https://user-images.githubusercontent.com/17181582/166525590-7aad4155-9599-4c3c-a811-03f4a854e78d.png)


## Notes:
